File: Projects/Multithreading/app/src/main/java/com/ivettevaldez/multithreading/MainActivity.java
package com.ivettevaldez.multithreading;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.ivettevaldez.multithreading.home.MainFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.container, MainFragment.newInstance())
                    .commitNow();
        }
    }
}

File: Projects/UnitTesting/app/src/main/java/com/ivettevaldez/unittesting/unittestinginandroid/example14/MainActivity.kt
package com.ivettevaldez.unittesting.unittestinginandroid.example14

import android.app.Activity
import android.os.Bundle
import com.ivettevaldez.unittesting.R

class MainActivity : Activity() {

    var count: Int = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        start()
    }

    fun start() {
        count++
    }
}